config/dockerfiles/Dockerfile.Rust

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
```

Comment:
I see… I suppose I could try and vet Jimmy Cuadra.  …or…

As far as burden or overhead the Dockerfile would only need a version number changed for future updates.  The correct docker image installation will happen with just that.  Then they'd only need to change the version number in `lib/submission_runners/rust.rb` .  I could make it simpler by duplicating the grep style code from the `Rakefile` so it automatically gets the version number from the `Dockerfile.Rust` - then they'd only need to change a version number in one location for updates resulting in pretty much no overhead.